apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -717,6 +727,238 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
 
     };
 
+    /**
+     * The function returns the normalized format of locale.
```

Comment:
Let's also add the reason for normalizing locale value. I believe it's due to the same issue mentioned in https://github.com/wso2/identity-apps/pull/5980

Reply:
Fixed with 80ebd166cd95667b4388d7e6d79c6050b3877c44